config/settings/base.py

```diff
@@ -33,6 +33,14 @@
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+SECONDARY_DB_ALIAS = "default"
+
+if env.db("SECONDARY_DATABASE_URL", default=""):
+    SECONDARY_DB_ALIAS = "secondary"
+    DATABASES[SECONDARY_DB_ALIAS] = env.db("SECONDARY_DATABASE_URL")
```

Comment:
Nit: this would be easier to find if it were next to the other database line so all the databases defined are right next to each other. You can move the conditional settings above the database dict and do it all in one place.